app/src/main/res/layout/fragment_event_container.xml

```diff
@@ -17,7 +17,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_gravity="bottom"
-        android:background="?android:attr/windowBackground"
+        android:background="#fff"
```

Comment:
@android:color/white

However, Android does not recommend pure white as background. Should use #eee